tarantool/connection.py

```diff
@@ -1751,3 +1765,765 @@ def _check_features(self):
         features_list = [val for val in CONNECTOR_FEATURES if val in server_features]
         for val in features_list:
             self._features[val] = True
+
+    def crud_insert(self, space_name: str, values: Union[tuple, list], opts={}) -> CrudResult:
+        """
+        Inserts row through the `tarantool-crud <https://github.com/tarantool/crud#insert>`__ module.
```

Comment:
> tarantool-crud

That's definitely not the way to name it. It's either `tarantool/crud` or simply `crud`. Since we have a link here, I prefet the latter one: `crud`.

Reply:
Done